src/main/java/Alex/UserInputStorage.java

```diff
@@ -0,0 +1,199 @@
+package Alex;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.time.format.DateTimeParseException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.format.DateTimeFormatter;
+import java.util.Scanner;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class UserInputStorage {
+
+    public static final DateTimeFormatter TIMEFORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd HHmm");
```

Comment:
I like how the constant name is all uppercased but I am not sure if timeformatter is one word or two.